File: Users/joel/Development/continuum/reverse_string.rs
fn reverse_string(s: &str) -> String {
    let mut chars = s.chars();
    let mut reversed = String::new();
    while let Some(c) = chars.next() {
        reversed.push(c);
    }
    reversed
}

fn main() {
    let original_string = "Hello, World!";
    let reversed_string = reverse_string(original_string);
    println!("Original String: {}", original_string);
    println!("Reversed String: {}", reversed_string);
}

File: reverse_string.py
# Reverse a String

def reverse_string(s):
    return s[::-1]

# Test the function
if __name__ == "__main__":
    test_str = "hello"
    print(f"Original: {test_str}")
    reversed = reverse_string(test_str)
    print(f"Reversed: {reversed}")